src/components/internal/InternalInput.tsx

```diff
@@ -21,58 +20,67 @@ const DEFAULT_PROPS = {
   pasteAllowed: true,
   isValid: true,
   __internal__focus: false,
-};
+} as const;
 
-InternalInput.TYPES = TYPES;
-InternalInput.VARIANTS = VARIANTS;
-InternalInput.COLORS = COLORS;
-InternalInput.NUMERIC_REGEX = NUMERIC_REGEX;
-InternalInput.DECIMAL_REGEX = DECIMAL_REGEX;
-InternalInput.DEFAULT_PROPS = DEFAULT_PROPS;
+export type InternalInputTypes = "text" | "password" | "email" | "tel";
```

Comment:
How about:
```
const TYPES = ["text", "password", "email", "tel"] as const;

export type InternalInputTypes = (typeof TYPES)[number];
```
?

Reply:
Honestly I don't want to do that at all. I want to remove TYPES and do what other design systems do and either export an `enum` or just define the types as a union of strings and let typescript handle the rest.

Reply:
As long as we support the following, I'm happy:
1. JavaScript (no TypeScript) users
2. Being able to import all the available input types from somewhere, so we could create the type radio input on the playground page.